### PR TITLE
Update building-the-node-using-nix.md

### PR DESCRIPTION
**Pull request by CarlosLopezDeLara**
Build the cardano-node executable. No default configuration.

### CODE REVIEW

File: doc/getting-started/building-the-node-using-nix.md [hunk 1]
```diff
@@ -44,6 +44,10 @@ or run in in one go:
 nix run github:input-output-hk/cardano-node#mainnet/node
 ```
 
+To build the cardano-node executable:
```
Comment (deepfire) [inline doc/getting-started/building-the-node-using-nix.md:47]: I would have reworded this to underscore -- something like -- "to build just the cardano-node executable"

Comment (deepfire) [reply doc/getting-started/building-the-node-using-nix.md:47]: And it doesn't help that line 35 above doesn't really explain what it achieves.

We could reword it too, from:
> Once Nix is installed, log out and then log back in.
>
> After that you can build the full node package with Mainnet configuration as follows:

Comment (CarlosLopezDeLara) [reply doc/getting-started/building-the-node-using-nix.md]: Amended the commit with your suggestions, thanks Serge!

Comment (CarlosLopezDeLara) [reply doc/getting-started/building-the-node-using-nix.md]: There goes another amendment! =)

File: doc/getting-started/building-the-node-using-nix.md [hunk 2]
```diff
@@ -24,15 +24,16 @@ To improve build speed, it is possible to set up a binary cache maintained by IO
 optional**):
 ```
 sudo mkdir -p /etc/nix
-cat <<EOF | sudo tee /etc/nix/nix.conf
+cat <<EOF | sudo tee -a /etc/nix/nix.conf
```
Comment (disassembler) [inline doc/getting-started/building-the-node-using-nix.md:27]: Can we fix the caches in this file?;we no longer use iohk.cachix.org. and the substituter should be cache.iog.io now (but the key still has hydra in the name)

**Review by disassembler — CHANGES_REQUESTED** [review]

File: doc/getting-started/building-the-node-using-nix.md (hunk 2 above)
Comment (CarlosLopezDeLara) [reply doc/getting-started/building-the-node-using-nix.md:27]: Ok, I think I understood correctly. I updated the commit.

**Review by disassembler — APPROVED** [review]
